nautobot_circuit_maintenance/views.py

```diff
@@ -70,6 +70,20 @@ class CircuitMaintenanceBulkDeleteView(generic.BulkDeleteView):
     table = tables.CircuitMaintenanceTable
 
 
+class CircuitMaintenanceJobView(generic.ObjectView):
+    """Special View to trigger the Job to look for new Circuit Maintenances."""
+
+    queryset = models.CircuitMaintenance.objects.all()
+
+    def get(self, request, *args, **kwargs):
+        """Custom GET to run a the Job."""
+        class_path = (
+            "plugins/nautobot_circuit_maintenance.handle_notifications.handler/HandleCircuitMaintenanceNotifications/"
+        )
+
+        return redirect(f"/extras/jobs/{class_path}")
```

Comment:
Probably better to use `redirect(reverse("extras:job", kwargs={"class_path": class_path}))` here

Reply:
nicer!